x/consensus/keeper/keeper.go

```diff
@@ -76,10 +77,17 @@ func (k Keeper) UpdateParams(ctx context.Context, msg *types.MsgUpdateParams) (*
 	}
 
 	paramsProto, err := k.ParamsStore.Get(ctx)
+
+	var params cmttypes.ConsensusParams
```

Comment:
This change is required to support genesis i.e. consensus params are not yet set.